applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.hpp

```diff
@@ -29,6 +29,13 @@ namespace Kratos
 
 	class CrBeamElement3D2N : public Element
 	{
+	private:
+		//const values
+		static constexpr int number_of_nodes = 2;
+		static constexpr int dimension = 3;
+		static constexpr unsigned int local_size = number_of_nodes * dimension;
+		static constexpr unsigned int element_size = local_size * 2;

```

Comment:
As they are static member variables they should named like: `msElementSize`, `msLocalSize` etc where "m" is for member and "s" is for static.

Reply:
alright sounds logical

Reply:
So I will wait for this change